grype/db/v6/data.go

```diff
@@ -72,6 +72,8 @@ func KnownPackageSpecifierOverrides() []PackageSpecifierOverride {
 
 		// jenkins plugins are a special case since they are always considered to be within the java ecosystem
 		{Ecosystem: string(pkg.JenkinsPluginPkg), ReplacementEcosystem: ptr(string(pkg.JavaPkg))},
+		// Bitnami is a special case since it's not a language ecosystem but is a package type
+		{Ecosystem: "bitnami", ReplacementEcosystem: ptr(string(pkg.BitnamiPkg))},
```

Comment:
This isn't needed. Since `pkg.Bitnami` is the string `"bitnami"` anyway ( se https://github.com/anchore/syft/blob/main/syft/pkg/type.go#L16 ) this line is a no-op.